src/UnitTestSharedLibrary/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,5 @@
+﻿Axe.Windows.UnitTestSharedLibrary.TestCategory
```

Comment:
I'm a bit confused why this file is here. I didn't think we ran analyzers on test projects?

Reply:
Removed analyzers